requirements.txt

```diff
@@ -274,11 +272,10 @@ six==1.16.0
     #   pyjwkest
     #   python-dateutil
     #   social-auth-app-django
-    #   social-auth-core
     #   url-normalize
 social-auth-app-django==4.0.0
     # via -r requirements.in
-social-auth-core==4.0.2
+social-auth-core==4.0.3
```

Comment:
I wonder if we could bump this even more. Though I guess this is enough since it has the support for the new pyjwt.

Reply:
Perhaps we could. I just wanted to play sure. What we get if we upgrade higher, that I don't know.

Reply:
So if we stay for now at this 4.0.3?

Reply:
Fine by me! :+1: